src/events.ts

```diff
@@ -733,6 +733,10 @@ export interface CueEnterEvent extends Event {
    * The textual content of this subtitle.
    */
   text?: string;
+  /**
+   * Data URI for image data of this subtitle.
+   */
+  image?: string;
```

Comment:
Are we not constructing the images for performance reasons? Aka:
```suggestion
  image?: Image;
```